components/org.wso2.carbon.identity.api.user.approval/org.wso2.carbon.identity.rest.api.user.approval.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/approval/v1/core/factories/ApprovalEventServiceFactory.java

```diff
@@ -0,0 +1,16 @@
+package org.wso2.carbon.identity.rest.api.user.approval.v1.core.factories;
```

Comment:
Let's add the license header. 
Ref- https://somindagamage.medium.com/adding-and-using-wso2-copyright-license-in-intellij-idea-8d172ce8c420